### PR TITLE
merge ipa/develop

### PR DESCRIPTION
**Pull request by urecha**
One request to merge them all.
ipa/develop is the alternative develop branch, which has served its purpose (don't delete it yet though) and brings a pleathra of new and reworked features with it. See the added `changelog.md` file for a change log or the updated `readme.md` for the updated feature descriptions.

And if you happen to be the one merging this PR, please **_Don't delete the branch_**.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,9 +4,9 @@ name: e2e tests
 
 on:
   push:
-    branches: [ master, develop ]
+    branches: [ master, develop, ipa/develop ]
   pull_request:
-    branches: [ master, develop ]
+    branches: [ master, develop, ipa/develop ]
```
Comment (TimMaa) [inline .github/workflows/ci.yml:9]: I think running for the IPA branch can be removed again

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -20,9 +20,9 @@ jobs:
         uses: cypress-io/github-action@v2
         with:
           build: npm run build-linux
-          start: npm run start
-          wait-on: "http://127.0.0.1:8080"
-          wait-on-timeout: 20
+          # start: npm run start
+          # wait-on: "http://127.0.0.1:8080"
+          # wait-on-timeout: 20
```
Comment (TimMaa) [inline .github/workflows/ci.yml:25]: If not needed, please fully delete

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```
Comment (TimMaa) [inline CHANGELOG.md:7]: The changelog should be updated before do the merge and release it to production, do we want to do it here or for the merge to `master`?

File: cypress.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-    "baseUrl": "http://127.0.0.1:8080",
+    "baseUrl": "https://urecha.github.io/",
```
Comment (TimMaa) [inline cypress.json:2]: This shouldn't be pointing to your personal instance anymore

File: CHANGELOG.md (hunk 3 above)
Comment (urecha) [reply CHANGELOG.md:7]: For the merge to `master`, in the respective release-branch.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (urecha) [reply .github/workflows/ci.yml:25]: Is needed to run sensible CI outside the IPA context.

File: cypress.json (hunk 4 above)
Comment (urecha) [reply cypress.json:2]: Set back to localhost.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (urecha) [reply .github/workflows/ci.yml:9]: Will be removed.

**Pull request merged**